releases/v21.2.5.md

```diff
@@ -0,0 +1,150 @@
+- A bug has been fixed which caused errors in rare cases when trying to divide `INTERVAL` values by `INT4` or `INT2` values. [#75079][#75079]
+- Fixed a bug that could occur when a `TIMETZ` column was indexed, and a query predicate constrained that column using a < or > operator with a `TIMETZ` constant. If the column contained values with time zones that did not match the time zone of the `TIMETZ` constant, it was possible that not all matching values could be returned by the query. Specifically, the results may not have included values within one microsecond of the predicate's absolute time. This bug was introduced when the `TIMETZ` datatype was first added in 20.1. It exists on all versions of 20.1, 20.2, 21.1, and 21.2 prior to this patch. [#75172][#75172]
+- Fixed an internal error, "estimated row count must be non-zero", that could occur during planning for queries over a table with a `TIMETZ` column. This error was due to a faulty assumption in the statistics estimation code about ordering of `TIMETZ` values, which has now been fixed. The error could occur when `TIMETZ` values used in the query had a different time zone offset than the `TIMETZ` values stored in the table. [#75172][#75172]
+- Restore now inserts a system.namespace entry for synthetic public schemas. [#74759][#74759]
+- A bug has been fixed that caused internal errors in queries with set operations, like `UNION`, when corresponding columns on either side of the set operation were not the same. This error only occurred with a limited set of types. This bug is present in versions 20.2.6+, 21.1.0+, and 21.2.0+. [#75276][#75276]
+- Fixed SQL Activity pages crashing when a column was sorted by the 3rd time. [#75486][#75486]
+- Updated the `String()` function of `roleOption` to add a space on the role `VALID UNTIL`. [#75494][#75494]
+- In particular cases, some queries that involve a scan which returns many results and which includes lookups for individual keys were not returning all results from the table. [#75512][#75512]
+
+### Performance improvements
+
+- Allow rangefeed streams to use separate HTTP connection when `kv.rangefeed.use_dedicated_connection_class.enabled` setting is turned on.  Using separate connection class reduces the possibility of out of memory errors when running rangefeeds against very large tables.  The connection window size for rangefeed can be adjusted via `COCKROACH_RANGEFEED_INITIAL_WINDOW_SIZE` environment variable, whose default is 128KB. [#74456][#74456]
+- Incremental `BACKUP`s now use less memory to verify coverage of prior backups. [#74588][#74588]
+- The merging of incremental backup layers during `RESTORE` now uses a simpler and less memory intensive algorithm. [#74593][#74593]
+
+### Contributors
+
+This release includes 64 merged PRs by 30 authors.
+We would like to thank the following contributors from the CockroachDB community:
```

Comment:
"We would like to ..." through the list of CRDB team members should be deleted - I think there is a flag on the release notes generation script for this to avoid generating them in future.